_data/projects.yml

```diff
@@ -194,18 +173,26 @@ demo video</a> is available."
       site: "https://atomsofconfusion.com/"
       status: *starting
       description: "Within every confusing piece of software code
-  are small patterns that can lead developers  to outputs that radically
+  are small patterns that can lead developers to outputs that radically
 differ from what was intended. In studying these “atoms of confusion” we
-look to build a firm empirical foundation for reducing code confusion in
-software development and, thus, also reduce the frequency of buggy and
-malfunctioning programs."
+look to build a firm empirical foundation for reducing code confusion among
+software developers. In doing so, we hope to reduce the frequency of buggy and
+malfunctioning programs, save developers time and money on maintenance costs, and perhaps in the long run,
+re-engineer computer languages for improved developer comprehension."
       products: "The <a href=\"https://atomsofconfusion.com\">project website</a> provides
 background on our theory, studies, and analysis for this work.
 We make all of our <a href=\"http://atomsofconfusion.com/experiments.html\">study materials and anonymized data</a>
 openly available so that other researchers can replicate, validate, and
 build on our findings. Our results have been used to fix bugs in a variety
 of software projects, including the
-<a href=\"https://github.com/torvalds/linux/commit/7aa92c4229fefff0cab6930cf977f4a0e3e606d8\">linux kernel</a>."
+<a href=\"https://github.com/torvalds/linux/commit/7aa92c4229fefff0cab6930cf977f4a0e3e606d8\">linux kernel</a>".
+      landmarks: Two of the three papers written about this project’s work have
```

Comment:
I would just say "two" or "several" instead of "two of the three".  Yanyan and Martin have also written papers in this vein  

The rest of this section is nicely done.

Reply:
Fixed